Unix/base/oiomi.h

```diff
@@ -2237,6 +2387,42 @@ FILE_EVENT3(30213, trace_Selector_AddHandler_AlreadyThere_Impl, LOG_WARNING, PAL
 #endif
 FILE_EVENT3(30214, trace_Selector_RemoveHandler_NotThere_Impl, LOG_WARNING, PAL_T("Selector_RemoveHandler: selector=%p, handler=%p, name=%T NOT REGISTERED"), Selector *, Handler *, const TChar *)
 #if defined(CONFIG_ENABLE_DEBUG)
+#define trace_Stop_OMI() trace_Stop_OMI_Impl(__FILE__, __LINE__)
```

Comment:
I have a question about WARNING logs here, if someone stop/restart omi, it will log a WARNING, is it? will this be confused by user? just pick it up, thanks.

Reply:
Since starting and stopping is not a common event, wouldn't it make sense to log as WARNING?

Currently, we have no record of someone starting or stopping OMI, since default log level is WARNING.  

We have a new issue where customers are complaining that OMI didn't restart.  We don't even have any record in our logs that OMI restart was attempted.

Reply:
ok, hi @jeffaco , does WARNING make sense here?

Reply:
George and I discussed it. Note that we have precedence here, as [SCXCore](https://github.com/Microsoft/SCXcore) logs agent startup and shutdown in it's log files (it also logs version information). In this way, you have exact startup/shutdown times for the server.

Note that, while we see a lot of startup/shutdown events in our testing, in a normal customer case OMI would start at system boot and would run indefinitely until machine shutdown or OMI upgrade. Thus, regular customers should not see very many of these messages logged.

Ultimately, for diagnostic purposes an an unrelated issue, we wanted our logfile to show startup/shutdown events. Since the default log level is WARNING, these messages had to be WARNING as well to make it though the minimum log level filter.

Let me know if you have further questions.

Reply:
ok, got it, thanks.